challenges/lib/src/test/java/challenges/LibraryTest.java

```diff
@@ -7,8 +7,21 @@
 import static org.junit.Assert.*;
 
 public class LibraryTest {
-    @Test public void testSomeLibraryMethod() {
-        Library classUnderTest = new Library();
-        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
-    }
+ @Test
```

Comment:
Great work on the tests. Multiple tests + checking what happens if the array is empty!